lib/special_cases.php

```diff
@@ -161,6 +161,23 @@ function preg_replace($pattern, $replacement, $subject, int $limit = -1, int &$c
     return $result;
 }
 
+/**
+ * @param resource|null $dir_handle
+ * @return string|false
+ * @Deprecated
```

Comment:
```suggestion
 * @deprecated
```

I think if you put a capital letter, it might be picked by doctrine annotations as a class (not sure though...)